src/components/ManageTodolist.js

```diff
@@ -0,0 +1,59 @@
+/*
+   Author: Michael Babko
+   Date: March 2021
+
+   Renders all of the live Todolist items from the Todolist API.
+   Allows for creating of Todolist items.
+  
+   PROPS:
+      createAlert(message, type) - Function to call in order to display alerts
+*/
+
+import React from 'react';
+import useItems from '../hooks/useItems.js';
+
+import CreateTodolistModal from './CreateTodolistModal';
+
+const ManageTodolist = (props) => {
+   const [items] = useItems();
+
+   /**
+    * Displays all Todolist items from the TodolistAPI
+    * @returns {JSX[]} - The array of Todolist items to display that are in chronological order.
+    */
+   const renderItemsTable = () => {
```

Comment:
-0 Bad variable naming.

This isn't rendering a table, its rendering a list. Bad variable naming alongside comments can confuse a developer.